newtab.html

```diff
@@ -6,17 +6,26 @@
     <link rel="stylesheet" href="/aos-master/dist/aos.css" />
     <link rel="stylesheet" type="text/css" href="style.css">
     <script type="text/javascript" src="jquery-3.3.1.min.js"></script>
-    <script type="text/javascript" src="scripts.js"></script>
     <script src="/aos-master/dist/aos.js"></script>
+    <script type="text/javascript" src="scripts.js"></script>
```

Comment:
Even if your script doesn't rely on AOS, make it a habit to keep your own script below your dependencies.